bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxServerHandler.java

```diff
@@ -301,6 +301,10 @@ public void rate(String mac, String rateCommand) {
         }
     }
 
+    public void sleep(String mac, Duration sleepDuration) {
+        sendCommand(mac + " sleep " + String.valueOf(sleepDuration.toSeconds()));
```

Comment:
I am wondering, since the API supports seconds, if it would make sense to also support seconds instead of minutes.

For sure the normal use case would be a timer that you use for listening to a web radio until you fall asleep and there it is fine to enter some minutes. However, one could also think of using this in rules to play some seconds of an alarm sound and automatically turn off the player afterwards.

Do you think it would make sense to use seconds instead of minutes or would you say that my use-case is too artificial?

Reply:
No, it's a good question. I debated whether to use minutes or seconds. So, I bailed out on making the decision and I asked the user who requested the feature. :wink: He suggested minutes.

https://community.openhab.org/t/feature-request-squeezebox-binding-add-sleep-channel/117278/9

Reply:
I guess the other option would've been to use the `Number:Time` QuantityType. But since the binding doesn't use QuantityTypes anywhere else (at least I don't think it does), I decided against it.

Reply:
I assume you saw my selection of 1 day as the upper limit, which was a completely arbitrary selection on my part.

Reply:
Yes I saw the 1 day for the upper limit and thought that should be sufficient.

Well if the community thinks minutes is sufficient, then leave it like that. I just thought we can offer whats possible and not restrict it arbitrarily.